GestorDeEstoque/Controllers/UsuarioController.cs

```diff
@@ -0,0 +1,44 @@
+using GestorDeEstoque.Data;
+using GestorDeEstoque.DTOs;
+using GestorDeEstoque.Models;
+using GestorDeEstoque.Repositories;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GestorDeEstoque.Controllers
+{
+    [ApiController]
+    [Route("/usuario")]
+    public class LoginController : ControllerBase
+    {
+        private readonly ApplicationDbContext _context;
+        private UsuarioRepository _usuarioRepository;
+
+        public LoginController(ApplicationDbContext context, UsuarioRepository usuarioRepository)
+        {
+            _context = context;
+            _usuarioRepository = usuarioRepository;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> CriarUsuarioAsync([FromBody] UsuarioDTO usuario)
+        {
+            var usuarioExiste = await _usuarioRepository.BuscarUsuarioPorEmailAsync(usuario.Email);
```

Comment:
O nome `usuarioExiste` dá ideia de que o valor da variável é um `boolean`

```suggestion
            var usuarioExistente = await _usuarioRepository.BuscarUsuarioPorEmailAsync(usuario.Email);
```